src/pages/index.vue

```diff
@@ -225,7 +160,7 @@ const HomePage = {
     const searchTerm = ref('')
     const handleSearch = async () => {
       await store.dispatch(`${SEARCH}/${UPDATE_QUERY}`, {
-        q: searchTerm.value || featuredSearches[featuredSearchIdx.value].term,
+        q: searchTerm.value,
```

Comment:
We would still need the fallback string when the user hits search without typing anything?